pkg/flare/archive_dca.go

```diff
@@ -60,6 +68,9 @@ func createDCAArchive(fb flarehelpers.FlareBuilder, local bool, confSearchPaths
 	getMetadataMap(fb)               //nolint:errcheck
 	getClusterAgentClusterChecks(fb) //nolint:errcheck
 	getClusterAgentDiagnose(fb)      //nolint:errcheck
+	getAgentDaemonSet(fb)
```

Comment:
`//nolint:errcheck` needs to be added on each new line here, to avoid linter errors (errcheck linter)